src/style.css

```
@@ -1 +1,88 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  max-width: 1200px;
+  height: 60px;
+  left: 0;
+  top: 0;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin-right: 50px;
+  justify-content: space-between;
+}
+
+.ul {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav_link {
+  display: block;
+  padding: 23px 0;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  margin-right: 20px;
+  transition: color 0.4s;
+  text-transform: uppercase;
+}
+
+.nav-item {
+  position: relative;
+  list-style-type: none;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.last_link {
+  margin-right: 0;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: relative;
+  width: 100%;
+  height: 4px;
+  left: 0;
+  top: 18.5px;
+  background: #00acdc;
+  border-radius: 8px;
+  display: block;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0 10px 50px;
```

add this margin to some wrapper

```suggestion
  margin: 10px 0;
```